language/resource-api/README.md

```diff
@@ -143,6 +142,8 @@ software { php:
 }
 ```
 
+The provider should return a string containing all namevar values in a string matching one of the title patterns. This allows the runtime environment to present resources in a way familiar to the user.
```

Comment:
```suggestion
The provider should return a single formatted string representing all namevar values in such a way that it matches one of the title patterns. This allows the runtime environment to present resources in a way familiar to the user.
```
Now I get it - how about this suggestion?

Reply:
I've added the where and how bit, too.